app/src/main/res/values/styles.xml

```diff
@@ -37,4 +37,12 @@
         <item name="android:textColor">@color/colorText</item>
         <item name="android:fontFamily">@font/roboto</item>
     </style>
+
+    <style name="MyDialogTheme" parent="Theme.MaterialComponents.Dialog.Alert">
```

Comment:
`Theme.MaterialComponents.Light` is used inside `AppTheme`. It's better to extend `Theme.MaterialComponents.Light.Dialog.Alert` instead of `Theme.MaterialComponents.Dialog.Alert `

Reply:
By extending from `Theme.MaterialComponents.Light.Dialog.Alert` you just to change one property `colorAccent` instead of 4. 
Title most of the time is left black as https://material.io/design/components/dialogs.html# 

In perfect case we wouldn't even need custom theme for dialogs, but we have incorrect `colorAccent` set in out `AppTheme`, fixing it properly would take quite a lot of time and testing. So let's stick to custom dialog theme.

Reply:
Thank you for the tips. Still with that theme I had to change two elements for the same result :) Still code is shorter now.